src/GraphQL/Types/Composite/IAbstractGraphType.cs

```diff
@@ -30,6 +43,12 @@ public static bool IsPossibleType(this IAbstractGraphType abstractType, IGraphTy
             return false;
         }
 
+        /// <summary>
+        /// Returns the proper graph type for the specified object for this abstract graph type. If the abstract
+        /// graph type implements <see cref="IAbstractGraphType.ResolveType"/>, then this method is called to determine
+        /// the best graph type to use. Otherwise, <see cref="IObjectGraphType.IsTypeOf"/> is called on each possible
+        /// graph type supported by the abstract graph type to determine if a match can be found.
+        /// </summary>
         public static IObjectGraphType GetObjectType(this IAbstractGraphType abstractType, object value, ISchema schema)
```

Comment:
I think we can safely remove `schema` argument and two lines

```
 if (result is GraphQLTypeReference reference)
                result = schema.AllTypes[reference.TypeName] as IObjectGraphType;
```

since this method is called after schema initialization. Also see `//TODO` below.

Reply:
#2327

Reply:
ok